tests/Main/DateTimeOffsetTests.fs

```diff
@@ -37,9 +37,9 @@ let tests =
         |> equal "2014-09-11T16:37:02.000+00:00"
 
     testCase "DateTimeOffset from Year 1 to 99 works" <| fun () ->
-        let date = DateTimeOffset(1, 1, 1, 0, 0, 0, TimeSpan.Zero)
+        let date = DateTimeOffset(1, 1, 2, 0, 0, 0, TimeSpan.Zero)
```

Comment:
@alfonsogarciacaro Were the tests failing? 

I am asking because changes to Dates tests are always tricky. And in the past, the tests were configured to only work on a machine configured on the UK timeline or something like that.

Reply:
It fails in Japan timezone ;)

You're right, I should have taken care of the problem. But it seems to be a very obscure error (only?) happening when creating a date 100 AD in Japan timezone. I suspect it's a mix of the fact [we already have a workaround for dates with year <= 99](https://github.com/fable-compiler/Fable/blob/d4106ca232bed645ba63de601387c8847ee05a32/src/fable-library/DateOffset.ts#L96-L98) and that Japan timezone changed from +0918 (zulu time) to +0900 [at some point in history](https://bugs.chromium.org/p/chromium/issues/detail?id=867806). So I hope it doesn't really affect users (fingers crossed).

Reply:
I checked on my machine and it is working so let's keep it like that.